openpype/hosts/photoshop/plugins/publish/extract_review.py

```diff
@@ -49,34 +57,41 @@ def process(self, instance):
                 "stagingDir": staging_dir,
                 "tags": self.jpg_options['tags'],
             })
-
+            processed_img_names = img_list
         else:
             self.log.info("Extract layers to flatten image.")
             img_list = self._saves_flattened_layers(staging_dir, layers)
 
             instance.data["representations"].append({
                 "name": "jpg",
                 "ext": "jpg",
-                "files": img_list,
+                "files": img_list,  # cannot be [] for single frame
                 "stagingDir": staging_dir,
                 "tags": self.jpg_options['tags']
             })
+            processed_img_names = [img_list]
 
         ffmpeg_path = openpype.lib.get_ffmpeg_tool_path("ffmpeg")
 
         instance.data["stagingDir"] = staging_dir
 
-        # Generate thumbnail.
+        source_files_pattern = os.path.join(staging_dir,
+                                            self.output_seq_filename)
+        source_files_pattern = self._check_and_resize(processed_img_names,
```

Comment:
Resized temp files won't be cleaned up. They should be added to `context.data["cleanupFullPaths"]` or removed here when are not needed anymore.

Reply:
It is in "stagingDir", so they will be removed by "cleanup.py", I checked.